go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/operator-framework/api
 go 1.15
 
 require (
+	cuelang.org/go v0.2.2
```

Comment:
0.3 is much more stable, I would pull the latest release here

Reply:
Already changed this locally. Haven't pushed it up.